packages/components/src/cookies_form/index.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import Form from 'nebenan-form/lib/form';
+import FormGroup from 'nebenan-form/lib/form_group';
+import Checkbox from 'nebenan-form/lib/checkbox';
+
+import styles from './index.module.scss';
+
+
+const CookiesDetails = ({ text }) => (
+  <div className={styles.details}>{text}</div>
+);
+
+const CookiesForm = ({
+  className,
+  compactView,
+  defaultModel,
+  translations,
+  onSubmit,
+  ...rest
+}) => {
+  const handleValidSubmit = ({ tracking }) => {
+    onSubmit({
+      dismissed: true,
+      tracking,
+      marketing: tracking,
+      timestamp: Date.now(),
+    });
+
+    if (!tracking && defaultModel.tracking) {
```

Comment:
perhaps we should move reload also to the outside handler?

Reply:
will be duplicated in each fe repo